engine/src/test/java/org/entando/entando/web/page/PageControllerIntegrationTest.java

```diff
@@ -931,7 +931,86 @@ private PageRequest createPageRequest(String pageCode, String groupCode, String
         pageRequest.setParentCode(parentCode);
         return pageRequest;
     }
-
+    
+    @Test
+    void testValidatePage() throws Throwable {
```

Comment:
I think that the test name is not meaningful, can you try something like "shouldValidatorReturnServerErrorOverNullParameterTitlePageOwnerModel" ... if you agree.

Reply:
Method renamed and divided in several tests